examples/motoko_examples/quicksort/canisters/rust/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+mod quicksort;
+
+#[ic_cdk_macros::query]
+fn sort(xs: Vec<i128>) -> Vec<i128> {
```

Comment:
Let's consider using `ic_cdk::export::candid::Int`

Reply:
The example has been updated to use Ints. Marking this as resolved.